esp/services/WsDeploy/WsDeployPlugin.cpp

```diff
@@ -59,7 +59,7 @@ ESP_FACTORY IEspService * esp_service_factory(const char *name, const char* type
 
 ESP_FACTORY IEspRpcBinding * esp_binding_factory(const char *name, const char* type, IPropertyTree *cfg, const char *process)
 {
-   if (strcmp(type, "WsDeploySoapBinding")==0)
+   if (strieq(type, "WsDeploySoapBinding")||strieq(type, "wsdeploy_http"))
```

Comment:
Does this need a comment why http is also included here?

Reply:
The existing form for the name was inconsistent and therefor hard to automatically generate.  _http also more accurately reflects that these bindings cover all HTTP based protocol bindings not just SOAP.  I'll add comments.